docs/UserGuide.md

```diff
@@ -442,21 +441,24 @@ Example:
 * `delAppt i/S1234567A @d/2024-10-27 @t/1100-1200` will delete an appointment on `2024-10-27` from `1100` to `1200` for the patient with NRIC `S1234567A`.
 
 
-  [Back to Table of Contents](#table-of-contents)
+[Back to Table of Contents](#table-of-contents)
 
 ### Managing Medical Conditions
 
 #### Adding Medical Conditions : `addMedCon`
 
-Adds medical condition to an existing patient in MediBase3.
+Adds a medical condition or multiple medical conditions to an existing patient in MediBase3.
```

Comment:
How about 
`Add medical condition(s) to an existing patient in MediBase3.`

just to shorten it?